batches/BatchService.ts

```diff
@@ -1,27 +1,41 @@
 import axios from 'axios';
 import Batch from './Batch';
 
+interface trainerBatchResp {
+	batches: Batch[];
+	validYears: [];
```

Comment:
validYears array needs a type is it number[] or string[]

Reply:
have now added the number type